src/handler/handler.cpp

```diff
@@ -134,7 +134,11 @@ void Handler3D::PixelUnproject( View& view, GLprecision winx, GLprecision winy,
 {
     const GLint viewport[4] = {view.v.l,view.v.b,view.v.w,view.v.h};
     const pangolin::OpenGlMatrix proj = cam_state->GetProjectionMatrix();
+#if defined(__EMSCRIPTEN__)
+    glUnProject(winx, winy, winz, (const float*)Identity4d, proj.m, viewport, &Pc[0], &Pc[1], &Pc[2]);
```

Comment:
This cast is incorrect. Identity4f should be used instead - I think this is why navigation was so crazy before.